internal/app/manifest.go

```diff
@@ -34,7 +34,7 @@ type ManifestClient struct {
 }
 
 type ManifestClientInterface interface {
-	GetManifestLocal(sdkConfig hooks.SDKCLIConfig, hookExecutor hooks.HookExecutor) (types.SlackYaml, error)
+	GetManifestLocal(ctx context.Context, sdkConfig hooks.SDKCLIConfig, hookExecutor hooks.HookExecutor) (types.SlackYaml, error)
 	GetManifestRemote(ctx context.Context, token string, appID string) (types.SlackYaml, error)
```

Comment:
note: It's nice to see that both `GetManifestLocal(ctx, ...` and `GetManifestRemote(ctx, ...` now except context.